cmd/pebble/main.go

```diff
@@ -71,11 +72,26 @@ func main() {
 	wfeImpl := wfe.New(logger, db, va, ca, *strictMode)
 	muxHandler := wfeImpl.Handler()
 
+	if len(c.Pebble.ManagementListenAddress) > 0 {
+		go func() {
```

Comment:
I don't know Go that well. Is there a reason you're using an anonymous function here which you directly call, instead of just running this code directly? I've only seen this construct in JavaScript before, but I don't think Go is as horribly broken as JS to justify this construct :D

Reply:
Well it is not for the same purpose. Autocall anonymous functions in Javascript are used to isolate the context (variables, mainly) altered by the called function, to not leak them in the outer scope.

Here it is to create a goroutine to execute several actions, then the blocking call to `ListenAndServeTLS`. This way you can listen on several interfaces on parallel.

Reply:
Ah, now I notice the small little `go` in front of `func()`. I've never used/seen goroutines before, so sorry for missing that :)

Reply:
No problem!